EKF/mag_fusion.cpp

```diff
@@ -142,6 +203,19 @@ void Ekf::fuseMag()
 
 	}
 
+	// rotate magnetometer earth field state into body frame
```

Comment:
Wondering if it would make sense to auto generated the computation of `mag_innov` as well. I assume we could remove some duplicated computation.

Reply:
It's just a coordinate rotation so doesn't buy us much.